apps/aws-notebook/tf/run_autossh.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -ex
+set -o pipefail
+
+if [[ -n "$SANDBOX_SSH_DOMAIN_SUFFIX" ]];then
+  exit 0
+fi
+
+
+
+cat << EOF >> ~/.ssh/config
+Host *${SANDBOX_SSH_DOMAIN_SUFFIX}
+  ProxyCommand cs ssh-proxy %h:443
+  UserKnownHostsFile /dev/null
+  User owner
+  StrictHostKeyChecking no
+  HashKnownHosts no
+  IdentityFile /root/.config/crafting/sandbox/id_client
+EOF
```

Comment:
Recommend put all these into the command line options and avoid updating the config file.